iree/turbine/kernel/wave/scheduling/loop_reconstruction.py

```diff
@@ -667,8 +667,9 @@ def construct_pipelined_loop(
         visualize,
     )
 
-    # Remove the unpipelined reduction.
+    # Remove the unpipelined reduction and the corresponding subgraph
     reduction.graph.erase_node(reduction.fx_node)
```

Comment:
maybe use `CustomOp.erase` and handle subgraph removel there?

Reply:
I would prefer that too, but in the `erase` function of Iterate I do not have access to the trace, which owns the subgraphs.